rpc/backend/client_test.go

```diff
@@ -145,3 +145,13 @@ func RegisterBlockByHash(
 		Return(resBlock, nil)
 	return resBlock, nil
 }
+
+func RegisterBlockByHashError(client *mocks.Client, hash common.Hash, tx []byte) {
```

Comment:
@facs95 I'm still not happy with where the mocks registrations are located in the repo. I tried moving them to `rpc/backend/mocks` package, but since they are in a `xxx_test` file I can't call them from the 
`backend` package. 

Any idea how to improve the structure, or am I overthinking this?